### ISSUE TITLE
Fusion reactor meltdown

### ISSUE THREAD

**Issue by LemADEC** [turbodiesel4598/NuclearCraft#613]
As of NuclearCraft-2.18k-1.12.2, when a fusion reactor meltdown, it'll remove it's core and all electromagnets. This a bit extreme and confusion for players since this effect isn't documented in the wiki and there's not much hint on what happened.
Furthermore, for admins, it would help to have a log saying that the reactor did have a meltdown so we can help the related player fix their setup. Said log could dump the current parameters of the reactor.

To summarize:
- please update wiki with fusion reactor meltdown when overheating
- please add a log when reactor goes critical

**Comment by turbodiesel4598** [comment]
@LemADEC I'm not the one who writes the wiki - anyone can go ahead and update the information as they please :)
I'll add a config for a log, when if true, the server will be notified about meltdowns.

**Comment by turbodiesel4598** [comment]
Didn't add a config, but solid-fuel fission, fusion and MSR meltdowns will be logged ;)
